### ISSUE TITLE
Order of server start and handle mounts

### ISSUE THREAD

**Issue by chrisrhymes** [platformsh-templates/nextjs#62]
### Describe the bug

I am trying to understand the order of operations in a deploy, as it seems to be out of sync. Is this correct? 
 
1. Build hook - yarn install, yarn build and copy .next to temp directory
2. Start the next server
3. Deploy hook - copies mounts back over .next directory

If correct, this means that the server has started with an old version of the .next directory and it could look for a particularly named js file in .next/server/pages, but once the deploy hook runs and the mount is updated, the js file may not exist anymore or be renamed, and the server returns an error.  

### Include some logs

Error: Cannot find module '/app/.next/server/pages/[...path].js'

### Reproducing

We had a dynamic path `pages/[...path]/index.tsx` which was renamed `[[...path]]/index.tsx`. After deploying to platform.sh it was still looking for `[...path].js` rather than the new `[[...path]].js` file. 

### Expected behavior

Mount the directories before starting the server so the server has access to the latest and correct .next directory. 

### Your environment

```yaml
type: 'nodejs:14'

build:
  flavor: none
```

**Comment by apfelbox** [comment]
I have the same issue. I already kill and auto-restart the process, but this doesn't work reliably and therefore we have frequent deploys without assets.

**Comment by dottodot** [comment]
Yes seeing the same issue, seems very inconsistent and sometimes results in a broken site

**Comment by apfelbox** [comment]
@dottodot you can try this command, as it seems to work way more reliable for us: https://github.com/platformsh-templates/nextjs/pull/468

```
systemctl --user restart app
```
